index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="main"> 
+    <nav>
+        <label class="my-logo"> My Logo</label>
+    </nav>
+
+    <a href="#" class="my-menu">
+        <div></div>
+        <div></div>
+        <div></div>
+    </a>
+
+    </div>
+    
+    <div class="shape-container">
+        <div class="shape-1"></div>
+        <div class="shape-2"></div>
+    </div>
+    
+
+    <div class="headline">
+        <h2> I’m George
+            Glad to see you!</h2>
```

Comment:
- [ ] Kindly use `h1` tag instead of `h2` tag because this one is main title and in this case it better to use `h1` tag.[Check this link](https://www.reliablesoft.net/h1-tag/) for more info.